app/models/api/config.rb

```diff
@@ -0,0 +1,17 @@
+class Api::Config
```

Comment:
Yeah, /lib sounds good to me.

And omg I didn't know that was a thing - I wanted something like that in a follow up to this PR but if it's pretty simple to do that I can try throwing that in. Thanks @Fryguy !